integration/spark/gradle.properties

```diff
@@ -1,3 +1,4 @@
 jdk8.build=true
 version=0.4.0-SNAPSHOT
 spark.version=3.1.2
+org.gradle.jvmargs=-Xmx1G
```

Comment:
Gradle stops complaining about low memory and cache evictions after this change.